sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/AnalyzeActionsClient.java

```diff
@@ -805,11 +805,48 @@ private AbstractiveSummarizationTaskParameters getAbstractiveSummarizationTaskPa
             try {
                 return operationResult.onErrorMap(Utility::mapToHttpResponseExceptionIfExists);
             } catch (RuntimeException ex) {
-                return monoError(logger, ex);
+                return monoError(LOGGER, ex);
             }
         };
     }
 
+    private Function<PollingContext<AnalyzeActionsOperationDetail>, AnalyzeActionsOperationDetail>
+        activationOperationLanguageApiSync(Iterable<TextDocumentInput> documents, TextAnalyticsActions actions,
```

Comment:
We should reuse this on the async version of `beginAnalyzeActions` .
same for others.

Reply:
It may not be possible here due to the different sync and async calls,